sw/device/lib/dif/dif_pwrmgr.h

```diff
@@ -0,0 +1,499 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PWRMGR_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PWRMGR_H_
+
+/**
+ * @file
+ * @brief <a href="/hw/ip/pwrmgr/doc/">Power Manager</a> Device Interface
+ * Functions
+ */
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_warn_unused_result.h"
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
System includes before project includes. (Also, have we been putting the file comment before or after includes? We should check to make sure we've picked on that's consistent.)